packages/iris-grid/src/GotoRow.tsx

```diff
@@ -55,343 +57,362 @@ interface GotoRowProps {
+              <div className="goto-row-text">
+                of{' '}
+                {dh.i18n.NumberFormat.format(DEFAULT_FORMAT_STRING, rowCount)}
               </div>
-              {gotoValueError && (
-                <div className="text-danger">{gotoValueError}</div>
+              {gotoRowError && (
+                <div className="text-danger">{gotoRowError}</div>
               )}
             </div>
+            <div className="goto-row-close">
+              <Button kind="ghost" onClick={onClose}>
+                <FontAwesomeIcon icon={vsClose} style={{ marginRight: '0' }} />
+              </Button>
+            </div>
           </div>
-        )}
-      </>
-    </IrisGridBottomBar>
-  );
-}
+          {model.isSeekRowAvailable && (
+            <div className="goto-row-row">
+              <div
+                className={classNames('goto-row-wrapper', {
+                  'is-inactive': isGotoRowActive,
+                })}
+                onClick={() => setIsGotoRowActive(false)}
+                onFocus={() => setIsGotoRowActive(false)}
+                role="group"
+              >
+                <div className="goto-row-text">Go to value</div>
+                <div className="goto-row-input">
+                  <select
+                    className="custom-select"
+                    onChange={event => {
+                      const columnName = event.target.value;
+                      onGotoValueSelectedColumnNameChanged(columnName);
+                    }}
+                    value={gotoValueSelectedColumnName}
+                    aria-label="column-name-select"
+                  >
+                    {columns.map(column => (
+                      <option key={column.name} value={column.name}>
+                        {column.name}
+                      </option>
+                    ))}
+                  </select>
+                </div>
+
+                {renderValueInput()}
+
+                <div>
+                  <Button
+                    kind="ghost"
+                    disabled={gotoValue === ''}
+                    onClick={() => {
+                      onGotoValueSubmit(true);
+                    }}
+                  >
+                    <FontAwesomeIcon icon={vsArrowUp} />
+                  </Button>
+                  <Button
+                    kind="ghost"
+                    disabled={gotoValue === ''}
+                    onClick={() => {
+                      onGotoValueSubmit(false);
+                    }}
+                  >
+                    <FontAwesomeIcon icon={vsArrowDown} />
+                  </Button>
+                </div>
+                {gotoValueError && (
+                  <div className="text-danger">{gotoValueError}</div>
+                )}
+              </div>
+            </div>
+          )}
+        </>
+      </IrisGridBottomBar>
+    );
+  }
+);
+GotoRow.displayName = 'GotoRow';
 
+export type GotoRowFocus = { focus: () => void };
```

Comment:
Rather than defining this down here, define this above the component declaration and add the types to the `forwardRef` call.